addok/core.py

```diff
@@ -102,9 +102,9 @@ def str_distance(self):
         return self._scores.get('str_distance', [0.0])[0]
 
     @classmethod
-    def from_id(self, _id):
-        """Return a result from it's document id."""
-        return Result(keys.document_key(_id))
+    def from_hashid(self, hashid):
```

Comment:
I don't know if we want to keep `from_(hash)id`, especially if we remove https://github.com/addok/addok/issues/254 soon.